handlers/handlers.go

```diff
@@ -21,25 +27,78 @@ var reCaptcha = recaptcha.R{
 	Secret: os.Getenv("RECAPTCHA_SECRET_KEY"),
 }
 
+//the function for the minifying
+var m = minify.New()
+
+func compileTemplates(filenames ...string) (*template.Template, error) {
+
+	var tmpl *template.Template
+	for _, filename := range filenames {
```

Comment:
If you rename _filenames_ to _templatePaths_, don't forget to rename _filename_ to _templatePath_.